### DISCUSSION TITLE
ORY KRATOS WITH POSTGRESQL andDOcker compose

### DISCUSSION THREAD

**Discussion by Aditya51293** [ory/kratos#3114] [Q&A] [answered]
I am using Kratos with postgresql and tried to use the docker command sudo docker-compose -f quickstart.yml -f quickstart-selinux.yml -f quickstart-postgres.yml up --build --force-recreate
It is showing port is listening at 0.0.0.0:3000 but in browser I couldn't able to access welcome page of cratos using 4455 port

**Comment by vinckr** [comment]
Hello @Aditya51293,
did you follow the Quickstart guide? At which point exactly did it fail? 
You should get this in the terminal once its all running:
`kratos-kratos-selfservice-ui-node-1  | Listening on http://0.0.0.0:4455`

**Reply by Aditya51293** [reply]
Hi @vinckr 
Yes, actually through docker it was listening on 0.0.0.0:3000 but in browser it was showing 404 not found
And https://www.ory.sh/docs/kratos/guides/deploy-kratos-example through this if I want to bypass the SSL as we dont have domain is it possible to use without the cert command.

**Reply by Aditya51293** [reply]
Hi @vinckr 
Please help me on this.

**Reply by James3UK** [reply, marked as answer]
I got it to work with this, the underlying is listening on 3000 for some reason.
    ports:
      - '4455:3000'
